rfcs/react-components/components/dev-docs.md

```diff
@@ -0,0 +1,66 @@
+As a new developer to the Fluent repo, but long time Fluent consumer and fan, the supporting documentation has been a rough experience. This has forced me to randomize other members of the team which slows us both down. This RFC seeks to lay out a plan to modernize the Fluent dev docs.
+
+## Current issues:
+
+- Mixture of v6, 7, 8 and 9 specific docs.
+- Docs seemed to have stopped being updated in the early days of v9 development.
+- Mixture of process, set up, and reference docs.
+- RFC’s seem to be a large part of the Fluent process now, but there’s no clear way for an outsider to observe or contribute to them.
+- Use of Github’s wiki isn't the best technical solution.
+
+## Goals of the new system:
+
+- Be more accessible to developers in their work flow.
+- Help developers onboard faster without distracting core team members.
+- Provide technical documentation and guides for common patterns and practices that do not make sense to live in examples or code samples.
+- Focus attention on areas under active development (web and react components)
+
+## Non-Goals
+
+- Provide an ongoing communications platform for project statuses, road maps and planning.
+- Provide documentation relevant to other disciplines outside engineering.
+- Provide documentation around consuming the library. These should live externally in the Storybook.
+- Provide documentation around filing bugs or how to engage with the team.
+
+## Proposed changes
```

Comment:
What about process for documentation? Who should own documents for specific things? IMHO it's a good idea to update the documentation after you make a change that impacts them. Ex: we update our build or scaffolding process, that should include not just the changes but any documentation to explain the new normal.

Reply:
Good callout! We'll add some 'how to' docs along the way.